readthedocs/core/utils/__init__.py

```diff
@@ -34,12 +39,15 @@ def broadcast(type, task, args, kwargs=None):  # pylint: disable=redefined-built
         servers = getattr(settings, "MULTIPLE_APP_SERVERS", [default_queue])
     elif type in ['build']:
         servers = getattr(settings, "MULTIPLE_BUILD_SERVERS", [default_queue])
+    task_results = []
     for server in servers:
-        task.apply_async(
+        task_result = task.apply_async(
             queue=server,
             args=args,
             kwargs=kwargs,
         )
+        task_results += task_result
+    return task_results
```

Comment:
If not using `chord()`, this could be replaced by `celery.group()`, giving us a native grouping of tasks.